### ISSUE TITLE
Support for LVGL 9

### ISSUE THREAD

**Issue by lewis-ajh** [nikthefix/Lilygo_Support_T_Display_S3_Long_LVGL_Odometer#2]
Is there any intention to try to use LVGL 9 in this project or upcoming projects in the future? I have tried to port the lilygo demos to lvgl9 but I am experiencing a lot of issues, flush_cb seems to be called and giving flush ready signal to lvgl, but nothing is coming to screen, what is the best place to discuss this with other avid users of t display amoled.

**Comment by nikthefix** [comment]
@lewis-ajh 
Yes. I'll be updating all examples to use LVGL 9 asap. Check the LVGL migration document to see where the breaking changes are. Mostly syntax changes I think - at least as far as these demos are concerned..
